platform/node/test/ignores.json

```diff
@@ -41,10 +41,9 @@
   "render-tests/regressions/mapbox-gl-js#3548": "skip - needs issue",
   "render-tests/regressions/mapbox-gl-js#3682": "https://github.com/mapbox/mapbox-gl-js/issues/3682",
   "render-tests/regressions/mapbox-gl-native#7357": "https://github.com/mapbox/mapbox-gl-native/issues/7357",
-  "render-tests/runtime-styling/image-add-sdf": "skip - https://github.com/mapbox/mapbox-gl-native/issues/9847",
```

Comment:
There's probably a few other tests than can be enabled.

Reply:
Yes - `render-tests/regressions/mapbox-gl-js#3548`.